### PR TITLE
Add keras deeplabv3plus notebook

### PR DESCRIPTION
**Pull request by elad-c**
## Pull Request Description:
Add Keras DeeplabV3+ notebook for quantizing a semantic segmentation model for IMX500

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [x] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+   "metadata": {},
+   "source": [
+    "# DeeplabV3+ Semantic Segmentation Keras Model - Quantization for IMX500\n",
+    "\n",
+    "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb)\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "In this tutorial, we'll quantize the DeeplabV3+ model for semantic segmentation with MCT's post-training quantization techniques. The pretrained model was generated with this [repo](https://github.com/bonlime/keras-deeplab-v3-plus), after making the following changes:\n",
+    "\n",
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb:15]: The pretrained model was generated with this [repo] -->

The pretrained model is based on this repo [bonlime/keras-deeplab-v3-plus]

File: tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+    "1. input image size set to 320x320\n",
+    "2. expand_dims in Lambda layer replaced with a simple Reshape layer\n",
+    "3. tf.compat.v1.image.resize in Lambda layer replaced with a Resizing layer\n",
+    "4. Added argmax at model output"
+   ]
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb:20]: I think it better to give more high level explanation:
1. input image size set to 320x320
2. Incompatible layers were replaced
3. Argmax was added at the model output (why? maybe a few words...)

File: tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+   "cell_type": "markdown",
+   "source": [
+    "### Download Pascel VOC 2012 dataset"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "7a1038b9fd98bba2"
+  },
+  {
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb:56]: Maybe a few words before the code: in this step we download the dataset from ... into the current directory...

File: tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+    "from huggingface_hub import from_pretrained_keras\n",
+    "\n",
+    "model = from_pretrained_keras('SSI-DNN/keras_deeplabv3_plus_320')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Create dataset object"
+   ],
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb:98]: think to move this part into "utils"? let's talk..

File: tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb [hunk 5]
```diff
@@ -0,0 +1,342 @@
...
+   "id": "015e760b-6555-45b4-aaf9-500e974c1d86",
+   "metadata": {},
+   "source": [
+    "## Evaluation on Pascal dataset\n",
+    "\n",
+    "### Floating point model evaluation"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb:255]: A few words before the code... in this step we evaluate the floating point model on PascalVoc using tensorflow MeanIoU metric.., etc.

**Review by Idan-BenAmi — APPROVED** [review]
